x-pack/test/security_solution_endpoint_api_int/apis/resolver/tree_entity_id.ts

```diff
@@ -0,0 +1,375 @@
+/*
```

Comment:
This file was the original `tree.ts` file. The test helper functions were renamed.